src/keys-detective/compare-keys-to-files.ts

```diff
@@ -89,7 +96,8 @@ export function compareKeysToFiles({
         ...res,
         files: normalizedGlob(
           `${res.baseFilesPath}/${isGlobal ? '' : scope}/*.${fileFormat}`
-        ),
+        )
+        .filter(filterPathByLang(langsToProcess, scopeAndLangFromPathOption)),
```

Comment:
Seems like you can create this filter once and use it across the function, generally speaking you can create it once with the command config no?